content/nginx-one/getting-started.md

```diff
@@ -136,17 +136,43 @@ If you followed the [Installation and upgrade](https://docs.nginx.com/nginx-agen
 
 ## Enable NGINX metrics reporting
 
-In order for NGINX One Console to show specific traffic and system metrics, you need to enable the appropriate API on your NGINX data plane instances. The sections below provide step-by-step instructions for both NGINX Plus and NGINX Open Source (OSS).
+The NGINX One Console dashboard relies on APIs for NGINX Plus and NGINX Open Source Stub Status to report traffic and system metrics. The following sections include instructions on how to enable those metrics.
+<!--
+In order for NGINX One Console to show specific traffic and system metrics, you need to enable the appropriate API on your NGINX data plane instances. The sections below provide step-by-step instructions for both NGINX Plus and NGINX Open Source (OSS). -->
 
 ### Enable NGINX Plus API
 
-{{< include "/use-cases/monitoring/enable-nginx-plus-api.md" >}}
+<!-- {{< include "/use-cases/monitoring/enable-nginx-plus-api.md" >}} -->
```

Comment:
This statement "comments out" the "include", which would otherwise set up common content with https://docs.nginx.com/nginx-instance-manager/monitoring/overview-metrics/#f5-nginx-plus-metrics